aquarius/events/events_monitor.py

```diff
@@ -454,7 +454,7 @@ def get_last_processed_block(self):
                 try:
                     if self._es_instance.es.ping() is True:
                         break
-                except elasticsearch.exceptions.ElasticsearchException as es_err:
```

Comment:
I think these should be treated separately. If it is not an ElasticsearchException, please log another error.

Reply:
oh, ok then